Microsoft/microsoft-sentinel/ingest/parser.yml

```
@@ -0,0 +1,38 @@
+name: microsoft-sentinel
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+  - name: read_event_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.alertTime}}"
+        output_field: datetime
+  - name: set-common-fields
+  - name: set-sentinel-fields
+
+stages:
+  set-common-fields:
+    actions:
+      - set:
+          "@timestamp": "{{read_event_date.datetime}}"
+          event.kind: "{{json_event.message.alertKind}}"
```

The values for `event.kind` is restricted: https://www.elastic.co/guide/en/ecs/8.11/ecs-allowed-values-event-kind.html.
```suggestion
          event.kind: "alert"
```